content/ember-data/v3/default-serializer.md

```diff
@@ -0,0 +1,31 @@
+---
+id: ember-data:default-serializers
+title: Default serializers usage
+until: '4.0.0'
+since: '3.12.0'
+---
+## Default serializer derprecations
+### deprecate adapter.serializer and adapter.defaultSerializer fallbacks
+If no application and type specific serializer is specified, the store will lookup the default serializer from the type's adapter. This behavior is now deprecated. This behavior is deprecated in favor of explicitly defining a serializer for the type or application as a whole as described below.
+
+### -default serializer fallback in store.serializerFor
+When absolutely no serializer has been defined by the app, i.e. there is no model, application, or adapter serializer, the default (`-json-api`) serializer must be used. This behavior is deprecated in favor of explicitly defining a serializer for the type or application as a whole as described below.
+
+## clearing the deprecations
+More information about custom serializers can be found on the [ember.js/guides](https://guides.emberjs.com/release/models/customizing-serializers/#toc_customizing-serializers)
+
+If a specific model type requires custom serialization, a type-specific serializer can be created.. Defining a type-specific serializer will clear the first deprecation, but won't clear both deprecations unless a type-specific serializer is defined for every type of model used in the application. To do this, create an `app/serializers/[type].js` with the following:
+
+    import { JSONAPISerializer } from '@ember-data/serializers';
+
+    export default JSONAPISerializer.extend({/* custom behavior */ });
```

Comment:
Missing backticks and language for these code samples, and we should use class syntax e.g.

```ts
export default class UserSerializer extends JSONApiSerializer {
  /* custom behavior */
}
```